ClinGen/clincoded
# Connects to #1410. Connects to #1411. Connects to #1412. Connects to #1413. Connects to #1414. Connects to #1415. Connects to #1437. Connects to #1488.

**Technical notes:**
Except for #1488, all described tickets in this PR are part of a workflow going from viewing the classification matrix, saving the classification and to viewing the evidence summary. As a result, the testing steps will group the tickets together.

***

**Steps to test #1488:**
1. Go to ^/users/
2. Enter either 'Buchovecky' or 'Crowley' in the 'Filter table by' search box at the upper right hand corner.
3. Expect to see a filtered entry being returned for the user.

***

**Steps to test #1437, #1410, #1411, #1412, #1413, #1414, #1415:**
1. Create a new GDM.
2. Add 2-3 PMIDs to the GDM.
3. For each PMID in the GDM, add various types of evidence and score each one of them. For testing purpose, please have at least 3 scored proband evidence (with or without segregation), 2 segregation evidence without probands but with LOD scores, 2 scored case-control evidence, and 3 scored experimental evidence.
4. Upon having the minimum set of evidence added and scored, expect to see the "View Classification Matrix" button at the upper right hand corner instead of the "Generate Summary" button.
5. Click on the "View Classification Matrix" button and expect to be taken to the Classification Matrix page consisting of the score matrix and the classification form group (#1414).
6. Between the score matrix and the form group, also expect to see 2 instruction notes in yellow and blue color.
7. In the Classification form group, select and fill out all 4 form inputs (including the "Evidence Summary" - #1437) and click on the "Save" button.
8. Then expect to be taken to the non-editable Classification Matrix page consisting of the score matrix and view-only classification information filled out on the previous page.
9. At the bottom of this page, expect to see 3 buttons - "Record Curation page", "Edit Classification" and "View Evidence Summary".
10. Click on the "View Evidence Summary" button and expect to be taken to a new window/tab consisting of a classification header (#1410), a panel showing scored case-level proband evidence (#1411), a panel showing case-level segregation evidence, a panel showing score case-control evidence (#1412), a panel showing scored experimental evidence (#1413), and a "Print PDF" button at the bottom of the page (#1415).
11. Click on the "Print PDF" button and expect to bring up the browser print dialog box.
12. Click on the "Close" button of this Evidence Summary page and expect to see the window/tab being closed.
13. Now on the non-editable Classification Matrix page, click on the "Edit Classification" button and expect to be taken back to the Classification Matrix page with the editable form group.
